### DISCUSSION TITLE
New Idea Probably?

### DISCUSSION THREAD

**Discussion by PancakeMPEG** [KadeArchive/Kade-Engine#731] [Ideas]
It be a good idea if kade engine 1.5.1 added a mash option button in the options menu

**Comment by isakube** [comment]
GHOST TAPPING.

**Comment by ghost** [comment]
Hi - Please elaborate. The meaning of a "mash button" isn't clear.

**Reply by PancakeMPEG** [reply]
You know how hard charts like beethoven or like run ya know it would be hard if people who didnt play the game would spam keys like before

**Comment by Lucky-56** [comment]
basically an auto clicker ?

**Reply by PancakeMPEG** [reply]
no
